test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -227,6 +227,14 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	},
 	// --
 
+	// k8s.io/kubernetes/pkg/apis/events/v1beta1
+	gvr("events.k8s.io", "v1beta1", "events"): {
```

Comment:
Could you also check if the events are stored in a separate etcd instance? I don't know exactly where the code is, @lavalamp @jpbetz do you know?

Reply:
I think it's config dependent, but if there's a way to test default config I can do that.